espresso-tests/README.md

```diff
@@ -0,0 +1,28 @@
+# Espresso Migration
+
+This document contains details surrounding the test migrating an exsisting Arbitrum Orbit chain to use the nitro-espresso-integration, and be compatible with the Espresso network.
+
+## Running the test
+
+Run the test by navigating to nitro-testnode/espresso-tests and run ./migration-test.bash
+
+
+## Steps related to upgrade in production
+
+Many of the steps in the test are nearly identical to the steps needed to upgrade an orbit network to be compatible with the espresso network.
```

Comment:
I think the instructions on how to carry out the upgrade should go into a new README in `orbit-actions`, not this repo. They should also contain instructions to run deploy and execute all the actions required for the Espresso upgrade.

Let's link to this test from the new README in orbit-actions for the curious.

Reply:
I agree and I'm actively working on a README in that repo.